lib/classes/Assets.php

```diff
@@ -10,8 +10,8 @@ public function __construct()
 
     public function frontend()
     {
-        $assetsPath = get_stylesheet_directory() . '/assets';
-        $assetsUri = get_stylesheet_directory_uri() . '/assets';
+        $assetsPath = get_stylesheet_directory();
+        $assetsUri = get_stylesheet_directory_uri();
```

Comment:
These temp (query) variables don't seem necessary anymore and only makes this code less readable. Could you remove them?

Reply:
Hooray!